src/services/matchHistoryService.ts

```diff
@@ -104,27 +105,5 @@ export const submitMatch = async (
 
     body["entry.2043626966"] = extraNotes !== undefined ? extraNotes : "";
```

Comment:
can we add the "entry." part in the service as opposed to having the callers have to do it? it's common code, so it should live inside the service.

Reply:
I don't understand this. How is the entry common code? Entry numbers and formatting are unique for each caller

Reply:
the entry numbers are, but the formatting shouldn't be. The format of every google form entry (i think) is "entry.NNN" 
because of this, all callers just specify IDs and not the keys that the request body is expecting.
aka users pass in NNN not "entry.NNN"